src/s2wasm.h

```diff
@@ -1167,6 +1173,42 @@ class S2WasmBuilder {
         }
       }
     }
+    if (startFunction.size()) {
```

Comment:
I don't understand what this addition does. Isn't the `start` an already existing function? What kind of work do we need to do here?